server/mediaserver.go

```diff
@@ -78,6 +78,7 @@ var httpPushResetTimer = func() (context.Context, context.CancelFunc) {
 }
 
 type rtmpConnection struct {
+	initializing    chan error
```

Comment:
Since this channel is just used for signaling and the type of the value doesn't matter we could use a `chan struct{}`.